ImportPackDialog.gd

```diff
@@ -1,20 +1,15 @@
 extends FileDialog
 const FileHandler = preload("res://FileHandler.gd")
-var file_handler: FileHandler
-
-func _ready():
-	self.file_handler = FileHandler.new()
-	pass
 
 func _on_FileDialog_dir_selected(dir_path):
 	print("Selected folder:", dir_path)
-	var new_path: String = file_handler.protobin_data_folder.plus_file(dir_path.get_file())
-	file_handler.ensure_folder_existance(new_path)
+	var new_path: String = FileHandler.downloaded_markers_dir.plus_file(dir_path.get_file())
```

Comment:
As discussed offline, this crashes. Please be sure to test the code before submitting it.